src/main/java/application/bookstore/model/User.java

```diff
@@ -51,6 +51,9 @@ public class User implements UserDetails {
     )
     private Set<Role> roles;
 
+    @Column(name = "is_deleted", nullable = false)
+    private boolean isDeleted = false;
```

Comment:
Default value is false 
```suggestion
    private boolean isDeleted ;
```